csharp/RowOriented/ParquetRowWriter.cs

```diff
@@ -95,6 +95,19 @@ public void WriteRows(IEnumerable<TTuple> rows)
             }
         }
 
+        public void WriteRowSpan(ReadOnlySpan<TTuple> rows)
```

Comment:
I initially tried to add this as an override of the existing `WriteRows` method, but this is not source backwards-compatible, as any existing calls to `WriteRows` using an array would be ambiguous between the overload that takes an `IEnumerable` and this new method taking a `ReadOnlySpan`. Places where `WriteRows(array)` is used would need to be changed to `WriteRows(array.AsSpan())` to disambiguate and use the new method. So using a new method name seems like a better approach.